index.js

```diff
@@ -34,23 +35,33 @@ function formatBody(body) {
 }
 
 function logRequestResponse(req, proxyResponse, err) {
-    const fileName = `${req.method}-${req.originalUrl}`.replace(/\//g, "|");
-    fs.writeFileSync(`${__dirname}/requests/${fileName}.json`, JSON.stringify({
+    const mappingFileName = `${req.method}|${service}${req.originalUrl}`
+        .replace(/\//g, "_")
+        .replace(/\?/g, "-")
+        .replace("=", "|");
+    const bodyFileName = `${mappingFileName}__Body`;
+
+    fs.writeFileSync(`${__dirname}/requests/${mappingFileName}.json`, JSON.stringify({
         request: {
-            headers: req.headers,
             method: req.method,
-            path: req.originalUrl,
-            body: formatBody(req.rawBody)
+            url: `${service}${req.originalUrl}`
         },
         response: proxyResponse ? {
-            headers: proxyResponse.headers,
+            headers: {
+                "content-type": req.headers["content-type"]
```

Comment:
Lost the other response headers. And yeah, we shouldn't assume there's a content-type

Reply:
Yep, I'll add the regular headers back in